client/components/GlobalControls/GlobalControlsButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Button, AnchorButton } from '@blueprintjs/core';
+import { Button, IButtonProps, AnchorButton } from '@blueprintjs/core';
```

Comment:
I have no concerns with this set of changes, but what's the reasoning behind it?